package.json

```diff
@@ -71,7 +71,8 @@
     "**/ember-electron/**/got": "^11.8.6",
     "**/ember-cli/**/ansi-html": "^0.0.9",
     "**/ember-cli/testem/fireworm/async": "^2.6.4",
-    "**/desktop/@doyensec/electronegativity/winston/async": "^2.6.4"
+    "**/desktop/@doyensec/electronegativity/winston/async": "^2.6.4",
+    "**/@hashicorp/design-system-components/ember-stargate": "^0.6.0"
```

Comment:
Did this not work by just manually updating the `yarn.lock` file? I'm not seeing any reason to need the resolution in the lock file

Reply:
More details on why this was added in the first place can be found in this [slack msg](https://hashicorp.slack.com/archives/C7KTUHNUS/p1718644636513669)